com.unity.multiplayer.mlapi/Tests/Runtime/NetworkManagerHelper.cs

```diff
@@ -0,0 +1,285 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using NUnit.Framework;
+using MLAPI.SceneManagement;
+using MLAPI.Transports.UNET;
+
+namespace MLAPI.RuntimeTests
+{
+    /// <summary>
+    /// Helper class to instantiate a NetworkManager
+    /// This also provides the ability to:
+    /// --- instantiate GameObjects with NetworkObject components that returns a Guid for accessing it later.
+    /// --- add NetworkBehaviour components to the instantiated GameObjects
+    /// --- spawn a NetworkObject using its parent GameObject's Guid
+    /// Call StartNetworkManager in the constructor of your runtime unit test class.
+    /// Call ShutdownNetworkManager in the destructor  of your runtime unit test class.
+    ///
+    /// Includes a useful "BuffersMatch" method that allows you to compare two buffers (returns true if they match false if not)
+    /// </summary>
+    public static class NetworkManagerHelper
```

Comment:
can we have a PR just for this helper class to go into RuntimeTests asm?

Reply:
I am going to defer to my comment above regarding the RpcQueueHistoryFrame.cs as a single file PR.  If you reverted the NetworkManagerHelper class without reverting the unit tests...then the build would fail.  While it would be easier to dissect the PR, it would be more complicated to remove since there would be another PR that depended upon the suggested single file NetworkManagerHelper PR and as such both PRs would have to be reverted.  Now, the unit tests could be reverted and the NetworkManagerHelper would stand on its own... so there is that.  However, the whole purpose of that class was so I could write my unit tests in the way I did.  So, I am going to ask that we keep this all together in one PR to simplify any potential future reverting and to make anyone looking at the PR realize the changes made were specific to the unit tests (which is the primary reason for this PR).

Does this sound reasonable?

Reply:
yes but no :P